src/main/kotlin/no/nav/syfo/service/MerVeiledningVarselFinder.kt

```diff
@@ -43,4 +44,15 @@ class MerVeiledningVarselFinder(
             )
         }
     }
+
+    private fun isBrukerUnder67Ar(fnr: String): Boolean {
+        val storedBirthdate = databaseAccess.fetchFodselsdatoByFnr(fnr)
+        return if (storedBirthdate.isEmpty() || storedBirthdate[0].isNullOrEmpty()) {
+            log.info("[MerVeiledningVarselFinder] Mangler lagret fodselsdato, sjekker PDL pa nytt")
+            pdlConsumer.isBrukerYngreEnn67(fnr)
+        } else {
+            log.info("[MerVeiledningVarselFinder] Sjekker om person er under 67 ut fra lagret fodselsdato")
+            pdlConsumer.isFodselsdatoMindreEnn67Ar(storedBirthdate.first())
```

Comment:
Burde isFodselsdatoMindreEnn67Ar ligge i en util fil eller noe? Tenker at når den ligger inne i pdlConsumer så høres det ut som man sjekker PDL, men det er egentlig bare en intern funksjon for å sjekke fødselsdato

Reply:
Enig